### DISCUSSION TITLE
Lost the character save

### DISCUSSION THREAD

**Discussion by gued0nes** [thijsvanloef/palworld-server-docker#271] [Q&A]
I'm having an issue. When I restart the server using the command 'sudo docker compose restart`, my friend and I, who were online, lost the character save. When entering the session, the game prompts me to create another character. I did create one, and the world is showing all the places I've already been, but it seems like a completely new account from scratch. How can I resolve this? Thank you!

**Comment by thijsvanloef** [comment]
You might want to try this: https://github.com/thijsvanloef/palworld-server-docker/issues/205#issuecomment-1920529950
